cscs-checks/cuda/cuda_memtest_check.py

```diff
@@ -20,7 +20,8 @@ def __init__(self):
             'wget %s' % src_url,
             'tar -xzf cuda_memtest-1.2.3.tar.gz',
             'cd cuda_memtest-1.2.3',
-            'patch -p1 < ../cuda_memtest-1.2.3.patch'
+            'patch -p1 < ../cuda_memtest-1.2.3.patch',
+            'patch -p1 < ../cuda_memtest-1.2.3.patch2'
```

Comment:
Why don't you combine the two patches in one?

Reply:
I tried doing it but I could not make it work!

Reply:
That's strange. You still have the "vanilla" version in the test's resources, so you should be able to produce a single patch by comparing the latest version vs. the original one.

Reply:
Let me retry.